config/default.schema.json

```diff
@@ -542,7 +542,7 @@
 		},
 		"ALERT_STATUS_URL": {
 			"type": "string",
-			"default": null,
+			"default": "https://status.dbildungscloud.dev/",
 			"description": "The url of status message provider (should end without a slash)."
```

Comment:
I don't really know what it is and why it was changed but description says it should end without a slash.